diffbot_gazebo/launch/diffbot_view.launch

```diff
@@ -5,14 +5,14 @@
   <arg name="gui" default="true"/>
   <arg name="headless" default="false"/>
   <arg name="debug" default="false"/>
-  <arg name="world_name" default="$(find diffbot_gazebo)/worlds/corridor.world"/>
+  <arg name="world_name" default="corridor"/>
   <arg name="x_pos" default="-2.0"/>
   <arg name="y_pos" default="-0.5"/>
   <arg name="z_pos" default="0.0"/>
 
   <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
   <include file="$(find gazebo_ros)/launch/empty_world.launch">
-    <arg name="world_name" value="$(arg world_name)"/>
+    <arg name="world_name" value="$(find diffbot_gazebo)/worlds/$(arg world_name).world"/>
```

Comment:
I thought about changing this too. But this way all external worlds have to be placed inside `diffbot_gazebo/worlds/`. So I would rather keep it as it is although this means that the "global path" has to be provided to the `world_name` arg of the launch file.

Reply:
Do you think of using the world from a different package?

Reply:
Yes exactly. Leaving this as it is would allow us to use worlds from different packages. Although I agree, it is not very likely that one will do that. But I know there are a bunch of world files in `/usr/share/gazebo-11/worlds`. These come with the standard ROS installation and I think it might be useful to launch the simulation with some of these.

Reply:
OK, make sense. I was trying to think of a way to have this flexible with worlds from the package and from Gazebo but I couldn't find yet, a way of doing this.

Reply:
Yes this would be ideal, because I also don't quite like that now it needs the whole package path.